tests/conftest.py

```diff
@@ -61,3 +79,8 @@ def pytest_addoption(parser):
         default=os.getenv('REDASH_SERVER_URL', 'http://localhost:5000'),
         help="URL to the Redash Server",
     )
+    group.addoption(
+        '--verify-server-url',
+        action='store_true',
+        default=not os.getenv('VERIFY_SERVER_URL', 'false').lower() == 'false',
+        help='verify the server url.')
```

Comment:
Please move the closing bracket to the new line and add a trailing comma. This way we can easily add kwargs with a single line in a git diff.